daemons/ipa-kdb/ipa_kdb_principals.c

```diff
@@ -2861,19 +2881,25 @@ remove_virtual_str_attrs(krb5_context kcontext, krb5_db_entry *entry)
 {
     char *str_attr_val;
     krb5_error_code kerr;
+    const char *str_attrs[] = {
+        OPT_PAC_TKT_CHKSUM_STR_ATTR_NAME,
+        OPT_PAC_PRIVSVR_CHKSUM_STR_ATTR_NAME,
+        NULL};
+
+    for(int i = 0; str_attrs[i] != NULL; i++) {
```

Comment:
Variable declaration is not supported in for-loop parameters in C89. If FreeIPA aims to comply to C99, I guess this is fine.

Reply:
We already do not hold to C89, see `daemon/ipa-kdb/README` and `configure.ac`:

```
$ git grep C99
asn1/asn1c/asn_system.h:#include <inttypes.h>   /* C99 specifies this file */
asn1/asn1c/asn_system.h:#include <stdint.h>     /* SUSv2+ and C99 specify this file, for uintXX_t */
configure.ac:AC_PROG_CC_C99
daemons/ipa-kdb/README:- C99 for-loops are permitted (and encouraged)
daemons/ipa-kdb/README:- a restricted set of other C99 features are permitted

```

Variable declaration is in use everywhere else:
```
$ git grep 'for.*(int '|wc -l
38
```